vars/buildPlugin.groovy

```diff
@@ -139,7 +139,25 @@ def call(Map params = [:]) {
                   mavenOptions += 'clean install'
                   try {
                     boolean usingArtifactCachingProxy = false
-                    if (artifactCachingProxyEnabled) {
+
+                    // If the build concerns a pull request, check if there is "skip-artifact-caching-proxy" label applied in case the user doesn't wan't ACP
+                    boolean prLabelsContainSkipACP = false
+                    final String skipACPLabel = 'skip-artifact-caching-proxy'
+                    if (!env.BRANCH_IS_PRIMARY) {
+                      withCredentials([
+                        usernamePassword(credentialsId: 'github-app-infra', usernameVariable: 'GITHUB_APP', passwordVariable: 'GH_TOKEN')
+                      ]) {
+                        if (isUnix()) {
+                          prLabelsContainSkipACP = sh(script: 'gh pr view $CHANGE_URL --json labels | grep --ignore-case \'"skip-artifact-caching-proxy"\'', returnStatus: true) == 0
```

Comment:
As per https://github.com/jenkins-infra/packer-images/blob/5178d7d49d9f087a9d6adf05241f336c769c9fbf/provisioning/ubuntu-provision.sh#L397-L403, the command line `gh` is installed on All the Linux agents that can be used (AWS VM, Azure VM or Kuberenets Container agents) 👍